src/components/Notifications/Form/__tests__/IntegrationRecipientTypeahead.test.tsx

```diff
@@ -152,21 +165,22 @@ describe('src/components/Notifications/Form/IntegrationRecipientTypeAhead', () =
     });
 
     it('Integration recipients that have been previously used in the form are disabled', async () => {
-        // render(<IntegrationRecipientTypeahead
-        //     selected={ ref1 }
-        //     integrationType={ IntegrationType.WEBHOOK }
-        //     onSelected={ fn() }
-        // />, {
-        //     wrapper: getConfiguredWrapper(async () => [])
-        // });
-        // render(<IntegrationRecipientTypeaheadx
-        //     selected={ ref1 }
-        //     integrationType={ IntegrationType.WEBHOOK }
-        //     onSelected={ fn() }
-        // />, {
-        //     wrapper: getConfiguredWrapper(async () => [])
-        // });
-
-        // await waitForAsyncEvents();
-    })
+        const formikValues: Partial<BehaviorGroup> = {
+            actions: [{ integration: ref1 }] as ActionIntegration[]
+        };
+
+        render(<IntegrationRecipientTypeahead
+            selected={ undefined }
+            integrationType={ IntegrationType.WEBHOOK }
+            onSelected={ fn() }
+        />, {
+            wrapper: getConfiguredWrapper(undefined, formikValues)
+        });
+
+        await waitForAsyncEvents();
+        userEvent.click(screen.getByRole('button', { name: /Options menu/i }));
+
+        await waitForAsyncEvents();
+        expect(screen.getAllByRole('option')[0].className).toEqual('pf-c-select__menu-item pf-m-disabled');
```

Comment:
Could we check if the following attribute is present? `aria-disabled` (and set to true) instead of the className?

Reply:
Unfortunately, I could not find a cleaner way to check if it was disabled. It does not have an aria-disabled attribute and when I used toBeDisabled, it also was complaining.

Reply:
Yes, they usually don't really disable it and they add a `aria-disabled`.
Maybe this is a bug in patternfly. Could you create an issue here? 
https://github.com/patternfly/patternfly-react